src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/ExternalUserSearchControllerIntTest.kt

```diff
@@ -0,0 +1,62 @@
+package uk.gov.justice.digital.hmpps.manageusersapi.resource
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.manageusersapi.integration.IntegrationTestBase
+
+class ExternalUserSearchControllerIntTest : IntegrationTestBase() {
+
+  @Nested
+  inner class FindUsersByEmail {
```

Comment:
is there any 'isNotFound' scenarios here?

Reply:
The end point in external users mimics the existing behaviour of the corresponding Auth end point, which gives a no content response when no match is found, and also when an email is not supplied